R/model.R

```diff
@@ -47,3 +47,40 @@ run_simulation <- function(pop, parameters = NULL, max_age = 100) {
   output
 
 }
+
+#' @title Run the simulation with repetitions
+#'
+#' @param repetitions n times to run the simulation
+#' @param overrides a named list of parameters to use instead of defaults
+#' @param parallel execute runs in parallel
+#' @return data frame for runs
+#' @export
+run_simulation_replicate <- function(
+  repetitions,
+  overrides = list(),
+  parallel = FALSE
+) {
+
+  # Currently running sequentially only
+  fapply <- lapply
+
+  # Currently running sequentially only
+  counter <- 0
+
+  dfs <- fapply(
+
+    seq(repetitions),
+    function(repetition) {
+      counter <<- counter + 1
```

Comment:
with the changes you made this works fine with indexing by `repetition`, as expected. can you please index by that (and drop the `counter` variable entirely) as this would prevent parallelisation in future